.github/workflows/cicd.yml

```diff
@@ -170,6 +189,25 @@ jobs:
       code_coverage: false
       configurations: '["NativeOnlyDebug", "NativeOnlyRelease"]'
 
+  driver_native_only_ws2022:
+    # Always run this job.
+    # Only run this on repos that have self-host runners.
+    needs: regular_native-only
+    if: github.repository == 'microsoft/ebpf-for-windows'
+    uses: ./.github/workflows/reusable-test.yml
+    with:
+      pre_test: .\setup_ebpf_cicd_tests.ps1
+      test_command: .\execute_ebpf_cicd_tests.ps1 -TestMode "CI/CD"
+      post_test: .\cleanup_ebpf_cicd_tests.ps1
+      name: driver
+      build_artifact: Build-x64-native-only
+      environment: ebpf_cicd_tests_ws2022
```

Comment:
update line 185 to say ws2019

Reply:
relabeling the existing 4 runners will require locking locking down the branch - otherwise, with this change merged all CI jobs will fail since there will be no corresponding runner.
And if I relabel prior to merging the change, same thing - existing yml won't match.

Reply:
Correction: I could do a staged rollout with no downtime due to re-labeling (same trick with extra 'migration' labels as last time), but renaming the existing runners will require some downtime.